packages/vue/src/components/organisms/SfCollectedProduct/SfCollectedProduct.vue

```diff
@@ -13,11 +13,9 @@
       >
     </slot>
     <slot name="more-actions">
-      <SfIcon
-        icon="more"
-        role="button"
-        class="sf-collected-product__more-actions mobile-only"
-      />
+      <button class="sf-collected-product__more-actions mobile-only">
```

Comment:
this button should have descriptive aria-label because there is no text inside. Please check other elements